browser/components/pictionary/PictionaryTimer.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import socket from '../../socket';
+
+import {connect} from 'react-redux';
+import { setTimer, countdown } from '../../reducers/timer';
+import { TIMER_DONE } from '../../../socketConstants';
+
+const mapStateToProps = state => ({
+  secondsRemaining: state.timer.secondsRemaining
+});
+
+const mapDispatchToProps = dispatch => ({
+  setTimer: startingSeconds => dispatch(setTimer(startingSeconds)),
+  countdown: () => dispatch(countdown()),
+});
+
+class PictionaryTimer extends Component {
+
+  componentWillMount() {
+    this.props.setTimer(60);
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(this.props.countdown, 1000);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.secondsRemaining === 0) {
```

Comment:
it would probably be good to add another clearInterval(this.interval) here so that there isn't a random timer still running in the background while the app waits for mobile to hit start